sabnzbd/deobfuscate_filenames.py

```diff
@@ -118,11 +118,11 @@ def is_probably_obfuscated(myinputfilename):
     return True  # default not obfuscated
 
 
-def deobfuscate(workingdirectory, usefulname):
-    """ In workingdirectory, check all filenames, and if wanted, rename """
+def deobfuscate_list(filelist, usefulname):
+    """ Check all files in filelist, and if wanted, deobfuscate """
 
```

Comment:
Just to be sure, I would add here at the start:
```python
filelist = [f for f in filelist if os.path.exists(f)]
```
Since all code below would crash the post-processor.
It could be possible that the paths in `newfiles` are invalid, since we parse them from unrar-output. Which could be ugly when running on Windows with Unicode.

Reply:
thanks